controlers/usersCtrl.js

```diff
@@ -26,10 +30,39 @@ const createUser = async (req, res, next) => {
     }
     try {
         const avatarURL = gravatar.url(email, { s: '200', r: 'pg', d: 'mm' });
-        const newUser = new User({email, password, avatarURL});
+        const verificationToken = uuidV4();
+        const newUser = new User({email, password, avatarURL, verificationToken});
         await newUser.setPassword(password);
         await newUser.save();
```

Comment:
najpierw wyślij email i jeśli wszystko pojdzie dobrze i nie będzie błędu to wtedy zapisz.